pkg/rke/k8s_rke_system_images.go

```diff
@@ -10352,6 +10352,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled in Rancher v2.9.1
```

Comment:
just to mention that those versions are actually released in the OOB release.

Reply:
Won't delay release for this as no changes in data.json

Reply:
@kinarashah @krunalhinguu , please fix the same problem on https://github.com/rancher/kontainer-driver-metadata/pull/1457